packages/utilities/src/crypto.ts

```diff
@@ -0,0 +1,68 @@
+import crypto from 'crypto';
```

Comment:
We are importing the `@apify/utilities` package on the frontend, so adding `crypto` as a dependency will cause the import to break, or we will require a huge polyfill. I would put these utilities in a separate package.

Reply:
Hm, OK, maybe it's not necessary to move then, tree shaking probably takes care of it. Let's keep it like this 🙂